package.json

```diff
@@ -2,14 +2,15 @@
   "name": "moviebox",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
   "scripts": {
-    "lint": "eslint backend",
-    "test": "npm run test:be:unit",
-    "server:dev": "./node_modules/.bin/webpack --config ./config/webpack.backend.config.babel.js --watch --hot",
-    "start": "DEBUG=moviebox nodemon ./dist/backend/server.js --watch ./dist/backend/",
-    "test:be:unit": "./node_modules/.bin/mocha --compilers js:babel-core/register \"test/unit/backend/**/*.spec.js\"",
-    "test:be:cover": "./node_modules/.bin/istanbul cover ./node_modules/mocha/bin/_mocha -- --compilers js:babel-core/register \"test/unit/backend/**/*.spec.js\""
+    "lint": "eslint backend config test",
+    "test": "npm run test:be:cover",
+    "test:unit": "npm run test:be:unit",
+    "test:cover": "npm run test:be:cover",
+    "server:dev": "webpack --config config/webpack.backend.base.config.babel.js --watch --hot",
+    "start": "DEBUG=moviebox nodemon dist/backend/server.js --watch dist/backend/",
+    "test:be:unit": "babel-node ./node_modules/.bin/_mocha test/unit/backend/**/*.spec.js",
+    "test:be:cover": "babel-node ./node_modules/.bin/babel-istanbul cover ./node_modules/.bin/_mocha -- test/unit/backend/**/*.spec.js"
```

Comment:
We do, because only `babel-node` is running on `package.json` context. Arguments for `babel-node` don't have the context anymore.